source/SIL.AppBuilder.Portal/common/workflow/state-machine.ts

```diff
@@ -720,11 +716,18 @@ export const WorkflowStateMachine = setup({
           }
```

Comment:
For the includeFields, you probably only need the listingLanguageCode for Android_GooglePlay.